CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# How to contribute to aleph
+
+This document is a work in progress!
+
+## Updating dependencies
+
+After you've made your changes to `project.clj`, be sure to run the tests with `lein test`!
+
+Once it's ready to go, you'll need to run the `deps/lein-to-deps` script to update `deps.edn` as well.
```

Comment:
Maybe mention that these generated changes should go in the same commit to make dependency updates atomic?